.github/workflows/build-and-test.yml

```diff
@@ -1,21 +1,17 @@
 name: Build and test from latest commit on master
-
 on:
   push:
     branches: [ master ]
   pull_request:
     branches: [ master ]
-
 jobs:
-
   build_and_test:
     name: Build and test on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [ubuntu-latest]
```

Comment:
Better to always have a locked/known version

Reply:
Locked on ubuntu-20.04